lnrpc/signrpc/signer.proto

```diff
@@ -332,23 +332,27 @@ message SignMessageReq {
     // Double-SHA256 hash instead of just the default single round.
     bool double_hash = 3;
 
+    // Don't hash the message before signing it. This option is only available
+    // when using a Schnorr signature.
+    bool no_hash = 4;
+
     /*
     Use the compact (pubkey recoverable) format instead of the raw lnwire
     format. This option cannot be used with Schnorr signatures.
     */
-    bool compact_sig = 4;
+    bool compact_sig = 5;
```

Comment:
We can't just re-number proto message fields, that would break backward compatibility.